source/common/common/logger.h

```diff
@@ -123,7 +124,66 @@ template <Id id> class Loggable {
 
 } // Logger
 
-#ifdef NDEBUG
+/**
+ * Base logging macros.  It is expected that users will use the convenience macros below rather than
+ * invoke these directly.
+ */
+#ifdef NVLOG
```

Comment:
Can we document this somewhere on the bazel readme page? (Basically what --copt to set, and what it does, useful for perf, etc.)